project_update_visible/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Also, you need to add the copyright information on all python files (except __init__.py files)
```suggestion
# Copyright 2023 Binhex (YOUR WEBSITE)
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).

{
```